sdks/python/apache_beam/coders/coders.py

```diff
@@ -201,12 +201,17 @@ def as_cloud_object(self):
         # We pass coders in the form "<coder_name>$<pickled_data>" to make the
         # job description JSON more readable.  Data before the $ is ignored by
         # the worker.
-        '@type': serialize_coder(self),
-        'component_encodings': list(
-            component.as_cloud_object()
-            for component in self._get_component_coders()
-        ),
+        '@type':
+            serialize_coder(self),
+        'component_encodings':
+            list(
```

Comment:
Here (and below) a list comprehension would be more natural.

Reply:
I'm not sure I understand your comment.  What are you asking for or suggesting?  (I'm not good at Python, so please be gentle and explicit.)  I didn't change this code except to add an argument, and since I'm so poor at Python, I'd prefer to avoid engaging in unrelated refactoring or clean-up.

Reply:
One would write `[expression for value in iterable]` rather than `list(expression for value in iterable)`. It's just something that struck me reading through the code; no need to do the refactoring now.

Reply:
got it, thanks.  i tried to make the right change.

Reply:
Spot on.